### DISCUSSION TITLE
Hubs Enterprise - Stack returns Hubs Cloud is Offline - Servers are failing the target group check with 500 errors.

### DISCUSSION THREAD

**Discussion by AS-Cellos** [mozilla/hubs#5847] [Hubs Cloud Questions]
**Description**
We are deploying a fresh Mozilla Hubs Cloud Enterprise Server in the Ireland Region with a non-route 53 domain name. We have setup the Certificate for both the North Virginia Region and Ireland Region, however when we deploy the cloud formation stack the servers are created and we map the cloudfront address to the sub-domain name set. It always resolves to hubs cloud is offline and the servers are returning a target group as unhealthy.

**Troubleshooting Currently**

We have deleted the stack and redeployed the stack with the same error.
We have terminated the servers within the stack to get the auto scaling group to restart new ones, and it returns the same error.
We have changed the server size from the defaulted selected one, and it returns the same error.
The stack that we deployed from is not from a previous backup, its a fresh install but it's just not resolving to the new domain name.
We have ran a CNAME check for DNS to see if the wildcard SSL Certificate has propagated world-wide and can see this.

**Expected behavior**

Expect the user login page to load for the server.

**Screenshots**
![Screenshot 2022-12-05 at 15 03 13](https://user-images.githubusercontent.com/53759306/205670735-d9db9f65-39e2-4919-ba7a-850bf38baf00.png)
<img width="865" alt="Screenshot 2022-12-05 at 15 01 14" src="https://user-images.githubusercontent.com/53759306/205670749-d59604dc-c9df-4f5f-9362-316b0a69fae5.png">

**Hardware** 
 - Device: Desktop
 - OS: All
 - Browser: All
 
**Additional context**

Please can you advise on any other troubleshooting steps that can be taken to understand why it's not resolving.

**Comment by pattersonbl2** [comment]
The two errors in the console are common for new hubs cloud deployments because they are not added until added in the admin panel. So what we need to focus on is the offline error.  This would be related to the configs breaking during instance start-up.  The only accurate way to know is by ssh into the ec2 and reviewing the logs. since you can't reach the admin panel to set up ssh auth.   I would look in your S3 buckets for ssh-totp.cfg file.  This would have the token id number that you can manually add to any auth app.
